kraken/pvc/pvc_scenario.py

```diff
@@ -0,0 +1,139 @@
+import sys
+import yaml
+import re
+import json
+import logging
+import time
+import kraken.cerberus.setup as cerberus
+import kraken.kubernetes.client as kubecli
+import kraken.invoke.command as runcommand
+
+# Reads the scenario config and creates a temp file to fill up the PVC
+
+
+def run(scenarios_list, config):
+    failed_post_scenarios = ""
+    for app_config in scenarios_list:
+        if len(app_config) > 1:
+            with open(app_config, "r") as f:
+                config_yaml = yaml.full_load(f)
+                scenario_config = config_yaml["pvc_scenario"]
+                pvc_name = scenario_config.get("pvc_name", "")
+                namespace = scenario_config.get("namespace", "")
+                target_fill_percentage = scenario_config.get("fill_percentage", "50")
+                duration = scenario_config.get("duration", 60)
+
+                # Check input params
+                if len(pvc_name) == 0:
+                    logging.error("You must specify the pvc_name")
+                    sys.exit(1)
+                if len(namespace) == 0:
+                    logging.error("You must specify the namespace where the PVC is")
+                    sys.exit(1)
+
+                # Get pod name
+                command = "oc describe pvc %s -n %s | grep \"Used By:\" | grep -Eo '[^: ]*$'" % (
```

Comment:
Maybe s/oc/kubectl to be able to run the scenario on kubernetes environments as well? Thoughts?

Reply:
Done. I change the `oc` commands to `kubectl` commands.